cves/CVE-2005-2700.yml

```diff
@@ -154,37 +163,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
-    note:
+    applies: true
+    note: mod_ssl (which is required) doesn't verify if virtaul host is set to optional.
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
     note:
   security_by_obscurity:
-    applies:
+    applies: false
     note:
   serial_killer:
-    applies:
+    applies: false
     note:
   environment_variables:
-    applies:
-    note:
+    applies: true
+    note: The bug is caused by a global configuration of the apache server.
```

Comment:
I'd always thought that the environment variables bit was referring to storing sensitive data in them, which may be visible to other processes